lib/search/query_components/core_query.rb

```diff
@@ -99,11 +99,12 @@ def match_phrase(field_name, boost)
       }
     end
 
-    def match_all_terms(fields)
+    def match_all_terms(fields, boost = 1.0)
```

Comment:
I guess this is added to give the 3 methods a similar signature.  I approve

Reply:
Actually it was added because I originally changed the boost here, but forgot to undo the change to the signature.  🤷‍♂